app/assets/stylesheets/custom.css.scss

```diff
@@ -225,3 +225,33 @@ a:hover, a:active {
     font: 400 18px/1 'Kaushan Script', cursive;
     color: $light-shadow-grey;
 }
+
+text {
+  font-size: 11px;
+  pointer-events: none;
+}
+
+text.parent {
+  fill: #1f77b4;
+}
+
+circle {
+  fill: #ccc;
```

Comment:
Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.